### PR TITLE
Fetch dachboard's data from MOOC

### PR DESCRIPTION
**Pull request by godu**
You can try to modify favorites playlist with this user : https://mobile-staging.coorpacademy.com/analytics/users/activity/user-5c751aa68f2713001b30e1b3

### CODE REVIEW

File: src/layer/data/cards.js [hunk 1]
```diff
@@ -1,40 +1,74 @@
...
+const LIMIT_CARDS = 5;
 
+export const fetchCards = async (
+  language: SupportedLanguage,
+  host: string,
+  token: string
+): Promise<Cards> => {
   if (__E2E__) {
-    disciplinesObject = {
+    let disciplinesObject: {[key: string]: Discipline} = {
```
Comment (VincentCATILLON) [inline src/layer/data/cards.js]: should be const as its not updated earlier

File: src/layer/data/cards.js [hunk 2]
```diff
@@ -1,40 +1,74 @@
...
-    disciplinesObject = {
-      ...onboarding.disciplines,
-      ...bescherelle.disciplines
-    };
-  }
 
-  // Implement fetching logic here
-  const disciplines = Object.keys(disciplinesObject).map(key => disciplinesObject[key]);
-  const cards = createDisciplinesCards(disciplines);
+    // Implement fetching logic here
```
Comment (VincentCATILLON) [inline src/layer/data/cards.js]: Comment can be removed as you did the fetching logic

File: src/layer/data/cards.js [hunk 3]
```diff
@@ -1,40 +1,74 @@
...
+  );
+  const result: {hits?: Cards} = await response.json();
+
+  return result.hits || [];
+};
+
+const LIMIT_CARDS = 5;
 
+export const fetchCards = async (
+  language: SupportedLanguage,
```
Comment (VincentCATILLON) [inline src/layer/data/cards.js]: 👍

File: src/redux/middlewares/cards.js [hunk 4]
```diff
@@ -20,6 +20,10 @@ import type {Cards, Level, Chapter} from '../../layer/data/_types';
 
 type State = StoreState;
 
+const TOKEN =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiNWM3NTFhYTY4ZjI3MTMwMDFiMzBlMWIzIiwiYnJhbmQiOiJtb2JpbGUiLCJob3N0IjoiaHR0cHM6Ly9tb2JpbGUtc3RhZ2luZy5jb29ycGFjYWRlbXkuY29tIiwiaWF0IjoxNTUxMTc4ODEwLCJleHAiOjIwMDAwMDAwMDAsImF1ZCI6Im1vYmlsZSIsImlzcyI6ImNvb3JwYWNhZGVteS1qd3QifQ.MRRlxll4lX5lel45679Nj-4IDGsVRwybxsUxbrl20Qs';
+const HOST = 'https://mobile-staging.coorpacademy.com';
```
Comment (VincentCATILLON) [inline src/redux/middlewares/cards.js]: This one is already defined in `brand-theme-provider` and should be used or mutualized (through redux?)

Comment (godu) [reply src/redux/middlewares/cards.js]: In the future, we'll need to store in redux the `(token, brand)` tuple. As you said, the card's middleware need to get host and token from store.
But i don't think it's a good idea to work on it before the authentification feature.

**Pull request merged**